### PR TITLE
Remove references of `shims` folder in docs [skip ci]

### PR DESCRIPTION
**Pull request by razajafri**
partially addresses issue #4920 

Signed-off-by: Raza Jafri <rjafri@nvidia.com>

### CODE REVIEW

**Review by revans2 — APPROVED** [review]

**Comment by razajafri** [comment]
I have tried to get rid of the CI requirement by pushing an empty commit but it seems like it's just easier to kick off the CI in this case instead fighting the system.

**Comment by razajafri** [comment]
build

**Review by gerashegalov — DISMISSED** [review]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -160,11 +160,9 @@ Spark version, open [Maven tool window](https://www.jetbrains.com/help/idea/2021
...
 if not triggered automatically.
 
-There is a known issue with the shims/spark3xx submodules. After being enabled once, a module such as shims/spark312
-may remain active in IDEA even though you explicitly disable the Maven profile `release312` in the Maven tool window.
-With an extra IDEA shim module loaded the IDEA internal build "Build->Build Project" is likely to fail
-(whereas it has no adverse effect on Maven build). As a workaround, locate the pom.xml under the extraneous IDEA module,
-right-click on it and select "Maven->Ignore Projects".
+There is a known issue where, even after selecting a different maven profile in maven submenu, the source folders from 
+previously selected profile may remain active. To get around this you have to manually reload the maven project from 
+the maven side menu
```
Comment (jlowe) [inline CONTRIBUTING.md:165]:
```suggestion
There is a known issue where, even after selecting a different Maven profile in the Maven submenu, the source folders from 
a previously selected profile may remain active. To get around this you have to manually reload the Maven project from 
the Maven side menu.
```

**Comment by razajafri** [comment]
build

**Review by jlowe — APPROVED** [review]

**Pull request merged**